dev/com.ibm.ws.security.token.ltpa_fat/fat/src/com/ibm/ws/security/token/ltpa/fat/LTPAInitializationVectorTests.java

```diff
@@ -0,0 +1,1476 @@
+    private static LibertyServer server2 = LibertyServerFactory.getLibertyServer("com.ibm.ws.security.token.ltpa.fat.initializationVectorTestServer2");
+
+    private static final Class<?> thisClass = LTPAKeyRotationTests.class;
+
+    // Initialize the user
+    private static final String validUser = "user1";
+    private static final String validPassword = "user1pwd";
+
+    private static final String[] serverShutdownMessages = { "CWWKS4102E", "CWWKG0058E", "CWWKG0083W", "CWWKS4106E", "CWWKS4109W", "CWWKS4110E", "CWWKS4111E", "CWWKS4112E",
+                                                             "CWWKS4113W",
+                                                             "CWWKS4114W", "CWWKS4115W", "CWWKS1859E" };
+
+    private static String validationKeyPassword = "{xor}Lz4sLCgwLTs=";
+    private static String validationKeyFIPSPassword = "{xor}CDo9Hgw=";
+
+    // Initialize the FormLogin Clients
+    private static FormLoginClient server1FlClient1;
+    private static FormLoginClient server1FlClient2;
+    private static FormLoginClient server2FlClient1;
+    private static FormLoginClient server2FlClient2;
+
+    // Define server.xml files
+    private static final String DEFAULT_SERVER_XML = "server.xml";
+    private static final String DEFAULT_FIPS_SERVER1_XML = "server1FIPS.xml";
+    private static final String DEFAULT_FIPS_SERVER2_XML = "server2FIPS.xml";
+
+    // Define the paths to the key files
+    private static final String DEFAULT_KEY_PATH = "resources/security/ltpa.keys";
+    private static final String CONFIGURED_VALIDATION_KEY1_PATH = "resources/security/configuredValidation1.keys";
+    private static final String FIPS_VALIDATION_KEY1_PATH = "resources/security/validation1.keys";
+    private static final String FIPS_VALIDATION_KEY2_PATH = "resources/security/validation2.keys";
+    private static final String VALIDATION_KEYS_PATH = "resources/security/";
+    private static final String VALIDATION_KEY1_PATH = "resources/security/validation1.keys";
+    private static final String VALIDATION_KEY2_PATH = "resources/security/validation2.keys";
+    private static final String VALIDATION_KEY3_PATH = "resources/security/validation3.keys";
+    private static final String BAD_SHARED_VALIDATION_KEY1_PATH = "resources/security/validation3.keys";
+    private static final String BAD_SHARED_VALIDATION_KEY2_PATH = "resources/security/validation4.keys";
+    private static final String BAD_PRIVATE_VALIDATION_KEY1_PATH = "resources/security/validation5.keys";
+    private static final String BAD_PRIVATE_VALIDATION_KEY2_PATH = "resources/security/validation6.keys";
+    private static final String BAD_PUBLIC_VALIDATION_KEY1_PATH = "resources/security/validation7.keys";
+    private static final String BAD_PUBLIC_VALIDATION_KEY2_PATH = "resources/security/validation8.keys";
+    private static final String DIFFERENT_PW_VALIDATION_KEY_PATH = "resources/security/validation9.keys";
```

Comment:
```suggestion
    private static final String DEFAULT_KEY_PATH = "resources/security/ltpa.keys";
    private static final String CONFIGURED_VALIDATION_KEY1_PATH = "resources/security/configuredValidation1.keys";
    private static final String FIPS_VALIDATION_KEY1_PATH = "resources/security/validation1.keys";
    private static final String FIPS_VALIDATION_KEY2_PATH = "resources/security/validation2.keys";
    private static final String VALIDATION_KEY1_PATH = "resources/security/validation1.keys";
    private static final String VALIDATION_KEY2_PATH = "resources/security/validation2.keys";
    private static final String VALIDATION_KEY3_PATH = "resources/security/validation3.keys";
    private static final String BAD_SHARED_VALIDATION_KEY2_PATH = "resources/security/validation4.keys";
    private static final String DIFFERENT_PW_VALIDATION_KEY_PATH = "resources/security/validation9.keys";
```

remove unused variables. Might be worth checking if the files are needed as well although i bet other tests use them